.teamcity/src/main/kotlin/configurations/SmokeTests.kt

```diff
@@ -22,6 +23,9 @@ class SmokeTests(model: CIBuildModel, stage: Stage, testJava: JvmCategory, task:
         extraParameters = buildScanTag("SmokeTests") +
             " -PtestJavaVersion=${testJava.version.major}" +
             " -PtestJavaVendor=${testJava.vendor.name}" +
-            " -Porg.gradle.java.installations.auto-download=false"
+            " -Porg.gradle.java.installations.auto-download=false",
+        preSteps = {
+            cleanAndroidUserHome()
```

Comment:
Are we running Android builds somewhere else as well? It seems to me that we would want to remove the file before every build.

Reply:
For me it looks like this is only a step for the builds that contain Android. I'm a bit reluctant to apply to all builds since some of the builds already have too many build steps.